Skeleton/src/Volunteer.cpp

```diff
@@ -1 +1,263 @@
-#include "../include/Volunteer.h";
+
+#include "../include/Volunteer.h"
+#include "../include/Order.h"
+
+
+/////////////////////////////////////////////////////////
+//////////////////////// Volunteer //////////////////////
+/////////////////////////////////////////////////////////
+
+Volunteer::Volunteer(int id, const string &name) : id(id), name(name) 
+{
+    completedOrderId = -1;
+    activeOrderId = -1;
+}
+
+Volunteer::Volunteer(int id, const string &name, int completedOrderId, int activeOrderId) : id(id), name(name), completedOrderId(completedOrderId), activeOrderId(activeOrderId) {}
+
+int Volunteer::getId() const
+{
+    return id;
+}
+
+const string& Volunteer::getName() const 
```

Comment:
return name as reference no value

Reply:
const refference

Reply:
Be aware that in the header file 'name' presented by value, not sure how to do it.
Please guide me if you know what needs to be done

Reply:
i think in the header fill it gives back string&
the signature must be the same, and if you need to give back const reference you should give it as the signature i think, even if its not mistake and it make auto its better to see it